functions/src/index.ts

```diff
@@ -44,22 +61,29 @@ interface IContainer {
   message_ts: string
   type: string
 }
+interface IActions {
+  action_id: string,
+  action_ts: string,
+  type:string
+}
 interface IBody {
   state: IState,
   response_url: string,
   user: IUser,
   container: IContainer,
   token: string
+  actions: IActions[]
 }
 const collectionName = "slack-status-update";
 export const myBot = functions.https.onRequest( async (req, res) => {
   if (req.body.command === "/make-status-update") {
     const modal = inputModal;
     res.send(modal);
     res.status(200);
-  } else {
-    const body: IBody = JSON.parse(req.body.payload);
-    console.log(body.token, config.slack.verification);
+    return;
+  }
+  const body: IBody = JSON.parse(req.body.payload);
```

Comment:
What will be if payload will be non-parsable?

Reply:
Done